Formula/node.rb

```diff
@@ -77,8 +78,8 @@ def install
       --shared-cares
```

Comment:
We might want to add `--openssl-use-def-ca-store` to use our CA store rather than the one that comes with Node.